charts/ccd-api-gateway-web/values.yaml

```diff
@@ -9,7 +9,7 @@ nodejs:
     CORS_ORIGIN_METHODS: GET,POST,OPTIONS,PUT,DELETE
     IDAM_SERVICE_NAME: ccd_gw
     SECURE_AUTH_COOKIE_ENABLED: true
-
+    UV_THREADPOOL_SIZE: 64
```

Comment:
this is already on develop: https://github.com/hmcts/ccd-api-gateway/pull/327/files

Reply:
any reason we're mixing different changes together?

Reply:
@MSancaktutar adding to above comments, are we sure we need this change here please?

Reply:
This is a performance improvement that could accompany the image upgrade.
The thread pool size increment may be particularly required by the new image. Even if not, it would be of no harm, wouldn't it?